src/components/parameters.js

```diff
@@ -48,13 +52,12 @@ const Parameters = () => {
     const classes = useStyles();
 
     const useName = useSelector(state => state.useName);
+    const centerLabel = useSelector(state => state.centerLabel);
+    const diagonalLabel = useSelector(state => state.diagonalLabel);
+    const [tabIndex, setTabIndex] = React.useState(useSelector(state => state.parametersTabIndex));
```

Comment:
Why does this come from the redux store ? To preserve between different router mount/unmounts ? But I don't see where the redux state is updated

Reply:
If we use the redux store, we shouldn't copy the state in this component I think ?